samples/cellular/nrf_cloud_multi_service/prj.conf

```diff
@@ -55,6 +55,8 @@ CONFIG_NET_IPV6_MLD=n
 CONFIG_NET_CONNECTION_MANAGER=y
 CONFIG_LTE_CONNECTIVITY=y
 CONFIG_NRF_MODEM_LIB_ON_FAULT_APPLICATION_SPECIFIC=y
+# Enable power savings mode
+CONFIG_LTE_PSM_REQ=y
```

Comment:
You know what, I'll bet we'll need to add an override for this in the Wi-Fi connectivity overlay

Reply:
Honestly the Wi-Fi connectivity overlay gets a lot of Kconfig warnings for other symbols in prj.conf, but I don't want to fix all that now.  I'll address this specific one though.